install/docker/opensuse-docker-install.md

```diff
@@ -5,13 +5,24 @@
 > - This guide assumes you have
 >   [already installed the Kata Containers packages](../opensuse-installation-guide.md).
 
-1. Install the latest version of Docker with the following commands:
-
-   > **Note:** This step is only required if Docker is not installed on the system.
+1. Install Docker with the following commands:
+
+   > **Notes:**
+   >
+   > - This step is only required if Docker is not installed on the system.
+   > - Newer versions of Docker have
+   >   [removed devicemapper support](https://github.com/kata-containers/documentation/issues/373)
+   >   so the commands below install the latest version which includes
```

Comment:
Suggested rewrite:
so the following commands install the latest version, which includes